### PR TITLE
MODULES-3816 no release 3.4.3 fixing changelog

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -266,15 +266,6 @@ kind of thing rather than trying to weave it into the main module logic itself.
...
 - Initdb should create xlogdir if set.
 
-##2014-08-27 - Supported Release 3.4.3
-###Summary
-
-This release fixes Ubuntu 10.04 with Facter 2.2.
-
-####Features
-####Bugfixes
-- Use a regular expression to match the major OS version on Ubuntu.
```
Comment (ferventcoder) [inline CHANGELOG.md]: This bit as well. Did this get moved into a different release changelog?

File: CHANGELOG.md [hunk 2]
```diff
@@ -266,15 +266,6 @@ kind of thing rather than trying to weave it into the main module logic itself.
 - Fix fedora support by using systemd.
 - Initdb should create xlogdir if set.
 
-##2014-08-27 - Supported Release 3.4.3
-###Summary
-
-This release fixes Ubuntu 10.04 with Facter 2.2.
```
Comment (tphoney) [inline CHANGELOG.md]: this is for a really old release, the fix has been in for a long time either way.

Comment (ferventcoder) [inline CHANGELOG.md]: That's fine, I'd just like to track where this change made it into an actual release.

**Comment by tphoney** [comment]
@ferventcoder i have moved the change/fix up into the next release.

**Pull request merged**